kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Environment.java

```diff
@@ -89,8 +89,9 @@ private Environment() {
      * The default value for skipping the teardown locally.
      */
     private static final boolean SKIP_TEARDOWN_DEFAULT = false;
+    public static final String KROXY_IMAGE_DEFAULT = KROXY_IMAGE_REPO_DEFAULT.split("/")[2];
+    public static final String KROXY_OPERATOR_IMAGE_DEFAULT = "operator";
```

Comment:
Why do you handle the default for the kroxylicious operand image and the operator image in a different way?

Reply:
because we don't have a default image repo for operator as it comes from the yaml file of the operator installation.

Reply:
You are much closer to the problem than me, but I _think_ I have the system tests unconditionally replace the image reference in the deployment.  That we we can point the system tests at any operator image and any operand image.

In my mind the system tests should have just two environment inputs:

KROXYLICIOUS_OPERATOR_IMAGE
KROXYLICIOUS_OPERAND_IMAGE

which default to quay.....:latest, so the system tests run from the IDE without change.
Jenkins PR builds both images and sets the two env vars.

Reply:
In general it is worth minimizing the number of switches/levers any system has. The number of paths through the system explodes and it makes (my) brain hurt. So. I think evolving to a simpler API would be desirable, in long term. (non-blocking).